iocore/net/SSLSNIConfig.cc

```diff
@@ -112,29 +114,25 @@ SNIConfigParams::SNIConfigParams() {}
 actionVector *
 SNIConfigParams::get(cchar *servername) const
 {
-  auto actionVec = sni_action_map.get(servername);
-  if (!actionVec) {
-    Vec<cchar *> keys;
-    wild_sni_action_map.get_keys(keys);
-    for (int i = 0; i < static_cast<int>(keys.length()); i++) {
+  auto action_it = sni_action_map.find(servername);
+  if (action_it != sni_action_map.end()) {
+    for (auto &&it : wild_sni_action_map) {
       std::string_view sv{servername, strlen(servername)};
```

Comment:
You don't need to do `strlen` explicitly, if it's a C-style string you can pass it as a `char const*`. E.g.
```
std::string_view sv{servername};
```

Reply:
As seen in other PRs, should this now be 

```
     for (auto const &it: wild_sni_action_map)
```

?